lib/memoizable/memory.rb

```diff
@@ -29,7 +29,7 @@ def initialize
     #
     # @api public
     def [](name)
-      @memory.fetch(name) do
+      fetch(name) do
```

Comment:
__comment:__ This seems like it could be a valid mutation, but mutant does not consider this an alive mutation.